### ISSUE TITLE
Inconsistent naming of `BlogLayout`/`BlogPostLayout`

### ISSUE THREAD

**Issue by ajcwebdev** [redwoodjs/learn.redwoodjs.com#121]
Part way through the tutorial we switch from `BlogLayout` to `BlogPostLayout` in the router.

### [Layouts](https://learn.redwoodjs.com/docs/tutorial/layouts)

>We'll wrap `HomePage` and `AboutPage` with the `BlogLayout`, using a `<Set>`:

```jsx
import { Router, Route, Set } from '@redwoodjs/router'
import BlogLayout from 'src/layouts/BlogLayout'

const Routes = () => {
  return (
    <Router>
      <Set wrap={BlogLayout}>
        <Route path="/about" page={AboutPage} name="about" />
        <Route path="/" page={HomePage} name="home" />
      </Set>
      <Route notfound page={NotFoundPage} />
    </Router>
  )
}

export default Routes
```

### [Routing Params](https://learn.redwoodjs.com/docs/tutorial/routing-params)

Code sample says one while the text explanation says the other:

>And while we're in the routes file, lets move the route inside the `Set` with the `BlogPostLayout`.

```jsx
<Router>
  <Set wrap={BlogLayout}>
    <Route path="/blog-post/{id}" page={BlogPostPage} name="blogPost" />
    <Route path="/about" page={AboutPage} name="about" />
    <Route path="/" page={HomePage} name="home" />
  </Set>
  <Route notfound page={NotFoundPage} />
</Router>
```

### [Web Authentication](https://learn.redwoodjs.com/docs/tutorial/authentication#web-authentication)

```jsx
import { Router, Route, Set, Private } from '@redwoodjs/router'
import BlogPostLayout from 'src/layouts/BlogPostLayout'

const Routes = () => {
  return (
    <Router>
      <Set wrap={BlogPostLayout}>
        <Route path="/blog-post/{id:Int}" page={BlogPostPage} name="blogPost" />
        <Route path="/contact" page={ContactPage} name="contact" />
        <Route path="/about" page={AboutPage} name="about" />
        <Route path="/" page={HomePage} name="home" />
      </Set>
      <Private unauthenticated="home">
        <Route path="/admin/posts/new" page={NewPostPage} name="newPost" />
        <Route path="/admin/posts/{id:Int}/edit" page={EditPostPage} name="editPost" />
        <Route path="/admin/posts/{id:Int}" page={PostPage} name="post" />
        <Route path="/admin/posts" page={PostsPage} name="posts" />
      </Private>
      <Route notfound page={NotFoundPage} />
    </Router>
  )
}

export default Routes
```

**Comment by clairefro** [comment]
Blasphemy! @cannikin which name you like better? `BlogLayout` seems to be logical

**Comment by cannikin** [comment]
Yeah should be `BlogLayout`, that other one may have snuck in when it got updated with the `<Set>`.

I like `BlogLayout` because it’s the layout for the whole blog, not just a single blog post.

**Comment by clairefro** [comment]
I'll make a quickie PR

**Comment by clairefro** [comment]
Resolved by #123
